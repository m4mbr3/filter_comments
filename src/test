N1 [20026_mb1_p5_02]
N2 [DATA: 08.04.2020   ORA: 11:34]
N3 T2 M6  [ UTENSILE DI FORATURA ]
N4 [ T2 CENTRINATURA ]
N5 [ CENTRI VITI M20 FIX CASTELLO ]
N6 S2000 M3 
N7 Z300 R 
N8 M8
N9 X305 Y-415 R 
N10 G81 Z54 J62 Q300 F50 
N11 X305 Y-415 
N12 X-10 
N13 X625 
N14 X1045 
N15 Y415 
N16 X620 
N17 X200 
N18 X-225 
N19 G80
N20 [ T2 CENTRINATURA ]
N21 [ CENTRI D30H7 X BUSSOLE CENTRAGGIO PIANO H37 ]
N22 Z300 R 
N23 X460 Y-415 R 
N24 G81 Z54 J62 Q300 F50 
N25 X460 Y-415 
N26 Y415 
N27 G80
N28 [ T2 CENTRINATURA ]
N29 [ CENTRI BUSSOLE D80H7 PIANO H227 ]
N30 Z300 R 
N31 X920 Y340 R 
N32 G81 Z224 J232 Q300 F50 
N33 X920 Y340 
N34 X735 Y-340 
N35 X-170 Y-380 
N36 X-70 Y380 
N37 G80
N38 [ T2 CENTRINATURA ]
N39 [ CENTRI VITI M10 FIX BUSSOLE PIANO H227 ]
N40 Z300 R 
N41 X866.739 Y309.25 R 
N42 G81 Z224 J232 Q300 F50 
N43 X866.739 Y309.25 
N44 X973.261 Y370.75 
N45 X788.261 Y-309.25 
N46 X681.739 Y-370.75 
N47 X-116.739 Y-349.25 
N48 X-223.261 Y-410.75 
N49 X-123.261 Y349.25 
N50 X-16.739 Y410.75 
N51 G80
N52 M5
N53 T1 M6  [ CILINDRICA ]
N54 [ T1 CONTORNATURA ]
N55 [ PROFILI SEDI BUSSOLE PIANO H52 OP20 ]
N56 S2000 M3 
N57 G49 K1
N58 Z300 R 
N59 M8
N60 X-175 Y250 R 
N61 G1 Z52 F50
N62 G41 D0=1 
N63 X-140.5 Y250 F200
N64 Y286.5 
N65 G3 X-160.5 Y306.5 I-160.5 J286.5 
N66 G1 X-189.5 
N67 G3 X-209.5 Y286.5 I-189.5 J286.5 
N68 G1 Y213.5 
N69 G3 X-189.5 Y193.5 I-189.5 J213.5 
N70 G1 X-160.5 
N71 G3 X-140.5 Y213.5 I-160.5 J213.5 
N72 G1 Y250 
N73 G40 
N74 X-175 Y250 
N75 Z300 R 
N76 X175 R 
N77 G1 Z52 F50
N78 G41 D0=1 
N79 X140.5 Y250 F200
N80 Y213.5 
N81 G3 X160.5 Y193.5 I160.5 J213.5 
N82 G1 X189.5 
N83 G3 X209.5 Y213.5 I189.5 J213.5 
N84 G1 Y286.5 
N85 G3 X189.5 Y306.5 I189.5 J286.5 
N86 G1 X160.5 
N87 G3 X140.5 Y286.5 I160.5 J286.5 
N88 G1 Y250 
N89 G40 
N90 X175 Y250 
N91 Z300 R 
N92 Y-240 R 
N93 G1 Z52 F50
N94 G41 D0=1 
N95 X140.5 Y-240 F200
N96 Y-276.5 
N97 G3 X160.5 Y-296.5 I160.5 J-276.5 
N98 G1 X189.5 
N99 G3 X209.5 Y-276.5 I189.5 J-276.5 
N100 G1 Y-203.5 
N101 G3 X189.5 Y-183.5 I189.5 J-203.5 
N102 G1 X160.5 
N103 G3 X140.5 Y-203.5 I160.5 J-203.5 
N104 G1 Y-240 
N105 G40 
N106 X175 Y-240 
N107 Z300 R 
N108 X-175 Y-250 R 
N109 G1 Z52 F50
N110 G41 D0=1 
N111 X-140.5 Y-250 F200
N112 Y-213.5 
N113 G3 X-160.5 Y-193.5 I-160.5 J-213.5 
N114 G1 X-189.5 
N115 G3 X-209.5 Y-213.5 I-189.5 J-213.5 
N116 G1 Y-286.5 
N117 G3 X-189.5 Y-306.5 I-189.5 J-286.5 
N118 G1 X-160.5 
N119 G3 X-140.5 Y-286.5 I-160.5 J-286.5 
N120 G1 Y-250 
N121 G40 
N122 X-175 Y-250 
N123 Z300 R 
N124 [ T1 CONTORNATURA ]
N125 [ PROFILI SEDI CHIAVETTE PIANO H142 OP20 ]
N126 Z300 R 
N127 X32 Y-130 R 
N128 G1 Z142 F50
N129 G41 D0=1 
N130 X41.75 Y-130 F200
N131 Y-93 
N132 G3 X22.25 Y-93 I32 J-93 
N133 G1 Y-167 
N134 G3 X41.75 Y-167 I32 J-167 
N135 G1 Y-130 
N136 G40 
N137 X32 Y-130 
N138 Z300 R 
N139 X22 Y25 R 
N140 G1 Z142 F50
N141 G41 D0=1 
N142 X22 Y34.75 F200
N143 X-15 
N144 G3 X-15 Y15.25 I-15 J25 
N145 G1 X59 
N146 G3 X59 Y34.75 I59 J25 
N147 G1 X22 
N148 G40 
N149 X22 Y25 
N150 Z300 R 
N151 X-5 Y130 R 
N152 G1 Z142 F50
N153 G41 D0=1 
N154 X4.75 Y130 F200
N155 Y167 
N156 G3 X-14.75 Y167 I-5 J167 
N157 G1 Y93 
N158 G3 X4.75 Y93 I-5 J93 
N159 G1 Y130 
N160 G40 
N161 X-5 Y130 
N162 Z300 R 
N163 M5
N164 T2 M6  [ UTENSILE DI FORATURA ]
N165 [ T2 CENTRINATURA ]
N166 [ CENTRI BUSSOLE D50H7 PIANO H52 OP20 ]
N167 S2000 M3 
N168 Z300 R 
N169 M8
N170 X175 Y-240 R 
N171 G81 Z49 J57 Q300 F50 
N172 X175 Y-240 
N173 X-175 Y-250 
N174 Y250 
N175 X175 
N176 G80
N177 [ T2 CENTRINATURA ]
N178 [ CENTRI FORI D70 X SEDE CILINDRI OP20 ]
N179 Z300 R 
N180 X130 Y0 R 
N181 G81 Z54 J62 Q300 F50 
N182 X130 Y0 
N183 X125 Y-90 
N184 X90 Y-240 
N185 X0 Y-265 
N186 X-90 Y-240 
N187 Y-90 
N188 X-130 Y0 
N189 X-120 Y90 
N190 X-90 Y240 
N191 X0 Y265 
N192 X90 Y240 
N193 X110 Y90 
N194 G80
N195 [ T2 CENTRINATURA ]
N196 [ CENTRI FORI D17 X VITI M6 FIX PUNZONE PIANO H152 OP20 ]
N197 Z300 R 
N198 X63 Y-130 R 
N199 G81 Z149 J157 Q300 F50 
N200 X63 Y-130 
N201 X0 Y-160 
N202 X32 Y145 
N203 X-48 Y155 
N204 G80
N205 [ T2 CENTRINATURA ]
N206 [ CENTRI SEDI DISTANZIALI PIANO D52 H52 OP20 ]
N207 Z300 R 
N208 X35.5 Y220 R 
N209 G81 Z54 J62 Q300 F50 
N210 X35.5 Y220 
N211 Y-220 
N212 G80
N213 [ T2 CENTRINATURA ]
N214 [ CENTRI VITI A COLLETTO M12 PIANO H52 OP20 ]
N215 Z300 R 
N216 X197 Y-292 R 
N217 G81 Z49 J57 Q300 F50 
N218 X197 Y-292 
N219 X-197 
N220 Y292 
N221 X197 
N222 G80
N223 [ T2 CENTRINATURA ]
N224 [ CENTRI VITI M8 FIX CHIAVETTE PIANO H142 OP20 ]
N225 Z300 R 
N226 X22 Y25 R 
N227 G81 Z139 J147 Q300 F50 
N228 X22 Y25 
N229 X-5 Y130 
N230 X32 Y-130 
N231 G80
N232 [ T2 CENTRINATURA ]
N233 [ CENTRI VITI M6 FIX BUSSOLE PIANO H52 OP20 ]
N234 Z300 R 
N235 X175 Y-200.5 R 
N236 G81 Z49 J57 Q300 F50 
N237 X175 Y-200.5 
N238 Y-279.5 
N239 X-175 Y-289.5 
N240 Y-210.5 
N241 Y210.5 
N242 Y289.5 
N243 X175 
N244 Y210.5 
N245 G80
N246 M5
N247 T1 M6  [ CILINDRICA ]
N248 [ T1 CONTORNATURA ]
N249 [ PROFILI SEDI BUSSOLE PIANO H52 OP30 ]
N250 S2000 M3 
N251 G49 K1
N252 Z300 R 
N253 M8
N254 X290 Y240 R 
N255 G1 Z52 F50
N256 G41 D0=1 
N257 X324.5 Y240 F200
N258 Y276.5 
N259 G3 X304.5 Y296.5 I304.5 J276.5 
N260 G1 X275.5 
N261 G3 X255.5 Y276.5 I275.5 J276.5 
N262 G1 Y203.5 
N263 G3 X275.5 Y183.5 I275.5 J203.5 
N264 G1 X304.5 
N265 G3 X324.5 Y203.5 I304.5 J203.5 
N266 G1 Y240 
N267 G40 
N268 X290 Y240 
N269 Z300 R 
N270 X625 R 
N271 G1 Z52 F50
N272 G41 D0=1 
N273 X590.5 Y240 F200
N274 Y203.5 
N275 G3 X610.5 Y183.5 I610.5 J203.5 
N276 G1 X639.5 
N277 G3 X659.5 Y203.5 I639.5 J203.5 
N278 G1 Y276.5 
N279 G3 X639.5 Y296.5 I639.5 J276.5 
N280 G1 X610.5 
N281 G3 X590.5 Y276.5 I610.5 J276.5 
N282 G1 Y240 
N283 G40 
N284 X625 Y240 
N285 Z300 R 
N286 Y-230 R 
N287 G1 Z52 F50
N288 G41 D0=1 
N289 X590.5 Y-230 F200
N290 Y-266.5 
N291 G3 X610.5 Y-286.5 I610.5 J-266.5 
N292 G1 X639.5 
N293 G3 X659.5 Y-266.5 I639.5 J-266.5 
N294 G1 Y-193.5 
N295 G3 X639.5 Y-173.5 I639.5 J-193.5 
N296 G1 X610.5 
N297 G3 X590.5 Y-193.5 I610.5 J-193.5 
N298 G1 Y-230 
N299 G40 
N300 X625 Y-230 
N301 Z300 R 
N302 X290 Y-240 R 
N303 G1 Z52 F50
N304 G41 D0=1 
N305 X324.5 Y-240 F200
N306 Y-203.5 
N307 G3 X304.5 Y-183.5 I304.5 J-203.5 
N308 G1 X275.5 
N309 G3 X255.5 Y-203.5 I275.5 J-203.5 
N310 G1 Y-276.5 
N311 G3 X275.5 Y-296.5 I275.5 J-276.5 
N312 G1 X304.5 
N313 G3 X324.5 Y-276.5 I304.5 J-276.5 
N314 G1 Y-240 
N315 G40 
N316 X290 Y-240 
N317 Z300 R 
N318 [ T1 CONTORNATURA ]
N319 [ PROFILI SEDI CHIAVETTE PIANO H149 OP30 ]
N320 Z300 R 
N321 X465 Y130 R 
N322 G1 Z149 F50
N323 G41 D0=1 
N324 X474.75 Y130 F200
N325 Y172 
N326 G3 X455.25 Y172 I465 J172 
N327 G1 Y88 
N328 G3 X474.75 Y88 I465 J88 
N329 G1 Y130 
N330 G40 
N331 X465 Y130 
N332 Z300 R 
N333 X480 Y0.001 R 
N334 G1 Z149 F50
N335 G41 D0=1 
N336 X480 Y-9.75 F200
N337 X522 
N338 G3 X522 Y9.75 I522 J0 
N339 G1 X438 
N340 G3 X438 Y-9.75 I438 J0 
N341 G1 X480 
N342 G40 
N343 X480 Y0.001 
N344 Z300 R 
N345 X482 Y-130 R 
N346 G1 Z149 F50
N347 G41 D0=1 
N348 X491.75 Y-130 F200
N349 Y-88 
N350 G3 X472.25 Y-88 I482 J-88 
N351 G1 Y-172 
N352 G3 X491.75 Y-172 I482 J-172 
N353 G1 Y-130 
N354 G40 
N355 X482 Y-130 
N356 Z300 R 
N357 M5
N358 T2 M6  [ UTENSILE DI FORATURA ]
N359 [ T2 CENTRINATURA ]
N360 [ CENTRI BUSSOLE D50H7 PIANO H52 OP30 ]
N361 S2000 M3 
N362 Z300 R 
N363 M8
N364 X625 Y-230 R 
N365 G81 Z49 J57 Q300 F50 
N366 X625 Y-230 
N367 X290 Y-240 
N368 Y240 
N369 X625 
N370 G80
N371 [ T2 CENTRINATURA ]
N372 [ CENTRI FORI D70 X SEDE CILINDRI OP30 ]
N373 Z300 R 
N374 X585 Y90 R 
N375 G81 Z54 J62 Q300 F50 
N376 X585 Y90 
N377 X545 Y235 
N378 X370 
N379 X335 Y90 
N380 X345 Y-90 
N381 X370 Y-240 
N382 X545 Y-235 
N383 X585 Y-90 
N384 G80
N385 [ T2 CENTRINATURA ]
N386 [ CENTRI FORI D17 X VITI M6 FIX PUNZONE PIANO H159 OP30 ]
N387 Z300 R 
N388 X480 Y25 R 
N389 G81 Z156 J164 Q300 F50 
N390 X480 Y25 
N391 X400 Y160 
N392 X520 
N393 X522 Y-145 
N394 X437 Y-170 
N395 G80
N396 [ T2 CENTRINATURA ]
N397 [ CENTRI SEDI DISTANZIALI PIANO D54 H52 OP30 ]
N398 Z300 R 
N399 X460 Y-260 R 
N400 G81 Z54 J62 Q300 F50 
N401 X460 Y-260 
N402 Y260 
N403 G80
N404 [ T2 CENTRINATURA ]
N405 [ CENTRI VITI A COLLETTO M12 PIANO H52 OP30 ]
N406 Z300 R 
N407 X268 Y282 R 
N408 G81 Z49 J57 Q300 F50 
N409 X268 Y282 
N410 X647 
N411 Y-282 
N412 X268 
N413 G80
N414 [ T2 CENTRINATURA ]
N415 [ CENTRI VITI M8 FIX CHIAVETTE PIANO H149 OP30 ]
N416 Z300 R 
N417 X480 Y0 R 
N418 G81 Z146 J154 Q300 F50 
N419 X480 Y0 
N420 X482 Y-130 
N421 X465 Y130 
N422 G80
N423 [ T2 CENTRINATURA ]
N424 [ CENTRI VITI M6 FIX BUSSOLE PIANO H52 OP30 ]
N425 Z300 R 
N426 X625 Y-190.5 R 
N427 G81 Z49 J57 Q300 F50 
N428 X625 Y-190.5 
N429 Y-269.5 
N430 X290 Y-279.5 
N431 Y-200.5 
N432 Y200.5 
N433 Y279.5 
N434 X625 
N435 Y200.5 
N436 G80
N437 M5
N438 T1 M6  [ CILINDRICA ]
N439 [ T1 CONTORNATURA ]
N440 [ PROFILO SEDE PIASTRA PIANO H212 OP40 ]
N441 S2000 M3 
N442 G49 K1
N443 Z300 R 
N444 M8
N445 X905 Y0 R 
N446 G1 Z212 F50
N447 G41 D0=1 
N448 X1140 Y0 F200
N449 Y262.5 
N450 X670 
N451 Y-262.5 
N452 X1140 
N453 Y0 
N454 G40 
N455 X905 Y0 
N456 Z300 R 
N457 [ T1 CONTORNATURA ]
N458 [ PROFILI SEDI RIFERIMENTI LAMIERA PIANO H202 OP40 ]
N459 Z300 R 
N460 X1105 Y119.999 R 
N461 G1 Z202 F50
N462 G41 D0=1 
N463 X1105 Y130.5 F200
N464 X1041.5 
N465 G3 X1041.5 Y109.5 I1041.5 J120 
N466 G1 X1105 
N467 G40 
N468 X1105 Y120.001 
N469 Z300 R 
N470 X975 Y274.25 R 
N471 G1 Z202 F50
N472 G41 D0=1 
N473 X964.5 Y274.25 F200
N474 Y247.5 
N475 G3 X985.5 Y247.5 I975 J247.5 
N476 G1 Y301 
N477 G3 X964.5 Y301 I975 J301 
N478 G1 Y274.25 
N479 G40 
N480 X975 Y274.25 
N481 Z300 R 
N482 X705 Y120.001 R 
N483 G1 Z202 F50
N484 G41 D0=1 
N485 X705 Y109.5 F200
N486 X786.5 
N487 G3 X786.5 Y130.5 I786.5 J120 
N488 G1 X705 
N489 G40 
N490 X705 Y119.999 
N491 Z300 R 
N492 Y-119.999 R 
N493 G1 Z202 F50
N494 G41 D0=1 
N495 X705 Y-130.5 F200
N496 X764.5 
N497 G3 X764.5 Y-109.5 I764.5 J-120 
N498 G1 X705 
N499 G40 
N500 X705 Y-120.001 
N501 Z300 R 
N502 X860 Y-280.25 R 
N503 G1 Z202 F50
N504 G41 D0=1 
N505 X870.5 Y-280.25 F200
N506 Y-253.5 
N507 G3 X849.5 Y-253.5 I860 J-253.5 
N508 G1 Y-307 
N509 G3 X870.5 Y-307 I860 J-307 
N510 G1 Y-280.25 
N511 G40 
N512 X860 Y-280.25 
N513 Z300 R 
N514 X1105 Y-120.001 R 
N515 G1 Z202 F50
N516 G41 D0=1 
N517 X1105 Y-109.5 F200
N518 X1033.5 
N519 G3 X1033.5 Y-130.5 I1033.5 J-120 
N520 G1 X1105 
N521 G40 
N522 X1105 Y-119.999 
N523 Z300 R 
N524 M5
N525 T2 M6  [ UTENSILE DI FORATURA ]
N526 [ T2 CENTRINATURA ]
N527 [ CENTRI VITI M16 FIX PIASTRA PIANO H212 OP40 ]
N528 S2000 M3 
N529 Z300 R 
N530 M8
N531 X786 Y-5 R 
N532 G81 Z209 J217 Q300 F50 
N533 X786 Y-5 
N534 Y-241.5 
N535 X1016 
N536 X1039 Y-25 
N537 Y241.5 
N538 X786 
N539 G80
N540 [ T2 CENTRINATURA ]
N541 [ CENTRI VITI M10 FIX RIFERIMENTI LAMIERA PIANO H202 OP40 ]
N542 Z300 R 
N543 X757.5 Y120 R 
N544 G81 Z199 J207 Q300 F50 
N545 X757.5 Y120 
N546 X748.5 Y-120 
N547 X860 Y-282.5 
N548 X1049.5 Y-120 
N549 X1057.5 Y120 
N550 X975 Y276.5 
N551 G80
N552 [ T2 CENTRINATURA ]
N553 [ CENTRI SPINE D16H7 POS PIASTRA PIANO H212 OP40 ]
N554 Z300 R 
N555 X790 Y-180 R 
N556 G81 Z209 J217 Q300 F50 
N557 X790 Y-180 
N558 X1005 Y235 
N559 G80
N560 [ T2 CENTRINATURA ]
N561 [ CENTRI SEDI CILINDRI X SOLLEVATORI PIANO D20 H187 OP40 ]
N562 Z300 R 
N563 X1018 Y0 R 
N564 G81 Z209 J217 Q300 F50 
N565 X1018 Y0 
N566 X1020 Y170 
N567 X805 
N568 X840 Y0 
N569 Y-170 
N570 X1020 
N571 G80
N572 [ T2 CENTRINATURA ]
N573 [ CENTRI FORI SCARICO D28 PIANO H212 OP40 ]
N574 Z300 R 
N575 X970 Y20 R 
N576 G81 Z209 J217 Q300 F50 
N577 X970 Y20 
N578 Y-170 
N579 X885 Y-65 
N580 X850 Y105 
N581 X940 Y170 
N582 G80
N583 M5
N584 M30
