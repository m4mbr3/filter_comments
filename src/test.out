N5  CENTRI VITI M20 FIX CASTELLO 
N21  CENTRI D30H7 X BUSSOLE CENTRAGGIO PIANO H37 
N29  CENTRI BUSSOLE D80H7 PIANO H227 
N39  CENTRI VITI M10 FIX BUSSOLE PIANO H227 
N55  PROFILI SEDI BUSSOLE PIANO H52 OP20 
N125  PROFILI SEDI CHIAVETTE PIANO H142 OP20 
N166  CENTRI BUSSOLE D50H7 PIANO H52 OP20 
N178  CENTRI FORI D70 X SEDE CILINDRI OP20 
N196  CENTRI FORI D17 X VITI M6 FIX PUNZONE PIANO H152 OP20 
N206  CENTRI SEDI DISTANZIALI PIANO D52 H52 OP20 
N214  CENTRI VITI A COLLETTO M12 PIANO H52 OP20 
N224  CENTRI VITI M8 FIX CHIAVETTE PIANO H142 OP20 
N233  CENTRI VITI M6 FIX BUSSOLE PIANO H52 OP20 
N249  PROFILI SEDI BUSSOLE PIANO H52 OP30 
N319  PROFILI SEDI CHIAVETTE PIANO H149 OP30 
N360  CENTRI BUSSOLE D50H7 PIANO H52 OP30 
N372  CENTRI FORI D70 X SEDE CILINDRI OP30 
N386  CENTRI FORI D17 X VITI M6 FIX PUNZONE PIANO H159 OP30 
N397  CENTRI SEDI DISTANZIALI PIANO D54 H52 OP30 
N405  CENTRI VITI A COLLETTO M12 PIANO H52 OP30 
N415  CENTRI VITI M8 FIX CHIAVETTE PIANO H149 OP30 
N424  CENTRI VITI M6 FIX BUSSOLE PIANO H52 OP30 
N458  PROFILI SEDI RIFERIMENTI LAMIERA PIANO H202 OP40 
N527  CENTRI VITI M16 FIX PIASTRA PIANO H212 OP40 
N541  CENTRI VITI M10 FIX RIFERIMENTI LAMIERA PIANO H202 OP40 
N553  CENTRI SPINE D16H7 POS PIASTRA PIANO H212 OP40 
N561  CENTRI SEDI CILINDRI X SOLLEVATORI PIANO D20 H187 OP40 
N573  CENTRI FORI SCARICO D28 PIANO H212 OP40 
N5  CENTRI VITI M20 FIX CASTELLO 
N21  CENTRI D30H7 X BUSSOLE CENTRAGGIO PIANO H37 
N29  CENTRI BUSSOLE D80H7 PIANO H227 
N39  CENTRI VITI M10 FIX BUSSOLE PIANO H227 
N55  PROFILI SEDI BUSSOLE PIANO H52 OP20 
N125  PROFILI SEDI CHIAVETTE PIANO H142 OP20 
N166  CENTRI BUSSOLE D50H7 PIANO H52 OP20 
N178  CENTRI FORI D70 X SEDE CILINDRI OP20 
N196  CENTRI FORI D17 X VITI M6 FIX PUNZONE PIANO H152 OP20 
N206  CENTRI SEDI DISTANZIALI PIANO D52 H52 OP20 
N214  CENTRI VITI A COLLETTO M12 PIANO H52 OP20 
N224  CENTRI VITI M8 FIX CHIAVETTE PIANO H142 OP20 
N233  CENTRI VITI M6 FIX BUSSOLE PIANO H52 OP20 
N249  PROFILI SEDI BUSSOLE PIANO H52 OP30 
N319  PROFILI SEDI CHIAVETTE PIANO H149 OP30 
N360  CENTRI BUSSOLE D50H7 PIANO H52 OP30 
N372  CENTRI FORI D70 X SEDE CILINDRI OP30 
N386  CENTRI FORI D17 X VITI M6 FIX PUNZONE PIANO H159 OP30 
N397  CENTRI SEDI DISTANZIALI PIANO D54 H52 OP30 
N405  CENTRI VITI A COLLETTO M12 PIANO H52 OP30 
N415  CENTRI VITI M8 FIX CHIAVETTE PIANO H149 OP30 
N424  CENTRI VITI M6 FIX BUSSOLE PIANO H52 OP30 
N458  PROFILI SEDI RIFERIMENTI LAMIERA PIANO H202 OP40 
N527  CENTRI VITI M16 FIX PIASTRA PIANO H212 OP40 
N541  CENTRI VITI M10 FIX RIFERIMENTI LAMIERA PIANO H202 OP40 
N553  CENTRI SPINE D16H7 POS PIASTRA PIANO H212 OP40 
N561  CENTRI SEDI CILINDRI X SOLLEVATORI PIANO D20 H187 OP40 
N573  CENTRI FORI SCARICO D28 PIANO H212 OP40 
N5 CENTRI VITI M20 FIX CASTELLO
N21 CENTRI D30H7 X BUSSOLE CENTRAGGIO PIANO H37
N29 CENTRI BUSSOLE D80H7 PIANO H227
N39 CENTRI VITI M10 FIX BUSSOLE PIANO H227
N55 PROFILI SEDI BUSSOLE PIANO H52 OP20
N125 PROFILI SEDI CHIAVETTE PIANO H142 OP20
N166 CENTRI BUSSOLE D50H7 PIANO H52 OP20
N178 CENTRI FORI D70 X SEDE CILINDRI OP20
N196 CENTRI FORI D17 X VITI M6 FIX PUNZONE PIANO H152 OP20
N206 CENTRI SEDI DISTANZIALI PIANO D52 H52 OP20
N214 CENTRI VITI A COLLETTO M12 PIANO H52 OP20
N224 CENTRI VITI M8 FIX CHIAVETTE PIANO H142 OP20
N233 CENTRI VITI M6 FIX BUSSOLE PIANO H52 OP20
N249 PROFILI SEDI BUSSOLE PIANO H52 OP30
N319 PROFILI SEDI CHIAVETTE PIANO H149 OP30
N360 CENTRI BUSSOLE D50H7 PIANO H52 OP30
N372 CENTRI FORI D70 X SEDE CILINDRI OP30
N386 CENTRI FORI D17 X VITI M6 FIX PUNZONE PIANO H159 OP30
N397 CENTRI SEDI DISTANZIALI PIANO D54 H52 OP30
N405 CENTRI VITI A COLLETTO M12 PIANO H52 OP30
N415 CENTRI VITI M8 FIX CHIAVETTE PIANO H149 OP30
N424 CENTRI VITI M6 FIX BUSSOLE PIANO H52 OP30
N458 PROFILI SEDI RIFERIMENTI LAMIERA PIANO H202 OP40
N527 CENTRI VITI M16 FIX PIASTRA PIANO H212 OP40
N541 CENTRI VITI M10 FIX RIFERIMENTI LAMIERA PIANO H202 OP40
N553 CENTRI SPINE D16H7 POS PIASTRA PIANO H212 OP40
N561 CENTRI SEDI CILINDRI X SOLLEVATORI PIANO D20 H187 OP40
N573 CENTRI FORI SCARICO D28 PIANO H212 OP40
