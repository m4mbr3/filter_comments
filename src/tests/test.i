%20026_mb1_p5_02G71
N1 % TNC:\INIZIO.H
N2 G30 G17 X-225 Y-415 Z49
N3 G31 X1140 Y415 Z300
N4 G90
N5 G247 Q339=1
* - UTENSILE DI FORATURA
N6 T2 G17 
* - OPERATION 1
* - T2 Centrinatura
N7 % TNC:\FINE.H
N8 % TNC:\INIZIO.H
* - CENTRI VITI M20 FIX CASTELLO
N9 S2000 M13 
N10 G00 X305 Y-415 
N11 Z300 
N12 G200 Q200=5 Q201=-3 Q206=50 Q202=100 Q210=0 Q203=57 Q204=243 Q211=0
N13 X305 Y-415 M99
N14 X-10 Y-415 M99
N15 X625 Y-415 M99
N16 X1045 Y-415 M99
N17 X1045 Y415 M99
N18 X620 Y415 M99
N19 X200 Y415 M99
N20 X-225 Y415 M99
