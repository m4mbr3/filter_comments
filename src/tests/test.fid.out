N1 CENTRI VITI M20 FIX CASTELLO
N14 CENTRI D30H7 X BUSSOLE CENTRAGGIO PIANO H37
N21 CENTRI BUSSOLE D80H7 PIANO H227
N30 CENTRI VITI M10 FIX BUSSOLE PIANO H227
N43 PROFILI SEDI BUSSOLE PIANO H52 OP20
N106 PROFILI SEDI CHIAVETTE PIANO H142 OP20
N142 CENTRI BUSSOLE D50H7 PIANO H52 OP20
N151 CENTRI FORI D70 X SEDE CILINDRI OP20
N168 CENTRI FORI D17 X VITI M6 FIX PUNZONE PIANO H152 OP20
N177 CENTRI SEDI DISTANZIALI PIANO D52 H52 OP20
N184 CENTRI VITI A COLLETTO M12 PIANO H52 OP20
N193 CENTRI VITI M8 FIX CHIAVETTE PIANO H142 OP20
N201 CENTRI VITI M6 FIX BUSSOLE PIANO H52 OP20
N214 PROFILI SEDI BUSSOLE PIANO H52 OP30
N277 PROFILI SEDI CHIAVETTE PIANO H149 OP30
N313 CENTRI BUSSOLE D50H7 PIANO H52 OP30
N322 CENTRI FORI D70 X SEDE CILINDRI OP30
N335 CENTRI FORI D17 X VITI M6 FIX PUNZONE PIANO H159 OP30
N345 CENTRI SEDI DISTANZIALI PIANO D54 H52 OP30
N352 CENTRI VITI A COLLETTO M12 PIANO H52 OP30
N361 CENTRI VITI M8 FIX CHIAVETTE PIANO H149 OP30
N369 CENTRI VITI M6 FIX BUSSOLE PIANO H52 OP30
N396 PROFILI SEDI RIFERIMENTI LAMIERA PIANO H202 OP40
N457 CENTRI VITI M16 FIX PIASTRA PIANO H212 OP40
N468 CENTRI VITI M10 FIX RIFERIMENTI LAMIERA PIANO H202 OP40
N479 CENTRI SPINE D16H7 POS PIASTRA PIANO H212 OP40
N486 CENTRI SEDI CILINDRI X SOLLEVATORI PIANO D20 H187 OP40
N497 CENTRI FORI SCARICO D28 PIANO H212 OP40
